examples/jupyter/import_config_pangeo.ipynb

```diff
@@ -0,0 +1,55 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "bf9da680",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from pan3d import DatasetBuilder"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2f052f51-86f1-4b44-bfa9-0c6d75892571",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "config_path = '../example_config_pangeo.json'\n",
+    "builder = DatasetBuilder(pangeo=True)\n",
```

Comment:
maybe the DatasetBuilder could also take a kwargs to for the UI creation too (ui=True)

Reply:
I added a `viewer` kwarg in [a2ed1f4](https://github.com/Kitware/pan3d/pull/67/commits/a2ed1f4423e5a78c798ab5668d4c4148efd700d3) that instantiates the viewer at the time the builder is instantiated. I updated the notebooks which need prior viewer instantiation to use this kwarg. We can discuss the name for this kwarg, but I don't want users to think that the ui is not accessible if they don't do this. Even if this kwarg is not specified, `builder.viewer` can still be accessed at any time (and will instantiate the Viewer class if one doesn't exist).

Reply:
I like what you pick.